docs/pages/database-access/guides/redis-cluster.mdx

```diff
@@ -0,0 +1,185 @@
+For example, if the first member is accessible at `redis1.example.com` with IP `10.0.0.1` and
+the second at `redis2.example.com` with IP `10.0.0.2`, run:
+```code
+$ tctl auth sign --format=redis --host=redis1.example.com,10.0.0.1 --out=redis1 --ttl=2190h
+$ tctl auth sign --format=redis --host=redis2.example.com,10.0.0.2 --out=redis2 --ttl=2190h
+```
+
+(!docs/pages/includes/database-access/ttl-note.mdx!)
+
+The command will create three files:
+- `server.cas` with Teleport's certificate authority
+- `server.key` with a generated private key
+- `server.crt` with a generated user certificate
+
+You will need these files to enable mutual TLS on your Redis server.
+
+(!docs/pages/includes/database-access/rotation-note.mdx!)
+
+Use the generated secrets to enable mutual TLS in your `redis.conf` configuration
+file and restart the database:
+
+```ini
+tls-port 7001
+port 0
+cluster-enabled yes
+tls-replication yes
+tls-cluster yes
+aclfile /path/to/users.acl
+masterauth GENERATED_STRONG_PASSWORD
+masteruser replica-user
+tls-cert-file /usr/local/etc/redis/certs/server.crt
+tls-key-file /usr/local/etc/redis/certs/server.key
+tls-ca-cert-file /usr/local/etc/redis/certs/server.cas
+tls-protocols "TLSv1.2 TLSv1.3"
+```
+
+Once mutual TLS has been enabled, you will no longer be able to connect to
+the cluster without providing a valid client certificate. You can use the
+`tls-auth-clients optional` setting to allow connections
+from clients that do not present a certificate.
+
+See [TLS Support](https://redis.io/topics/encryption)
+in the Redis documentation for more details.
+
+## Step 5/6. Create cluster
+
+Use the following command to create the cluster. Please note `redis-cli --cluster create` accepts only IP addresses.
+```sh
+export REDISCLI_AUTH=STRONG_GENERATED_PASSWORD
+export CERTS_DIR=/path/to/certs/
+export IP1=10.0.0.1 # update with the real node 1 IP
+export IP2=10.0.0.2 # update with the real node 2 IP
+export IP3=10.0.0.3 # update with the real node 3 IP
+export IP4=10.0.0.4 # update with the real node 4 IP
+export IP5=10.0.0.5 # update with the real node 5 IP
+export IP6=10.0.0.6 # update with the real node 6 IP
+redis-cli --user alice --cluster-replicas 1 --tls --cluster-yes \
+--cluster create ${IP1}:7001 ${IP2}:7002 ${IP3}:7003 ${IP4}:7004 ${IP5}:7005 ${IP6}:7006 \
+--cacert ${CERTS_DIR}/server.cas --key ${CERTS_DIR}/server.key --cert ${CERTS_DIR}/server.crt
+```
+
+## Step 6/6. Connect
+
+To enable Redis cluster mode in Teleport, add the `mode=cluster` parameter to the connection URI in
+your Teleport Database Service config file.
+```yaml
+databases:
+  - name: "redis-cluster"
+    uri: "rediss://redis.example.com:6379?mode=cluster"
+```
+
+(!docs/pages/includes/database-access/redis-connect.mdx!)
+
+### Supported Redis Cluster commands
+
+Redis in cluster mode doesn't support below commands. If one of the listed commands above is called Teleport
+returns the <nobr>`ERR Teleport: command not supported`</nobr> error.
+- `ACL`
```

Comment:
Can this long list be hidden under collapsible section? I think we have a "details" component for this.